frontend/src/components/Plan/MembersList.module.css

```diff
@@ -0,0 +1,15 @@
+
+.membersList {
+  margin: 1rem 1rem;
```

Comment:
careful using rem's. i think general advice is to use `px` over `rem` unless you know the distinction well

Reply:
let's track something in the tracker then to capture this